.github/workflows/main.yml

```diff
@@ -37,5 +37,6 @@ jobs:
         run: |
           $(go env GOPATH)/bin/ziti-ci configure-git
           $(go env GOPATH)/bin/ziti-ci tag -v -f version
-          $(go env GOPATH)/bin/ziti-ci trigger-github-build openziti/fabric update-dependency --token ${{ secrets.ZITI_CI_GH_TOKEN }}
+          $(go env GOPATH)/bin/ziti-ci trigger-github-build openziti/storage update-dependency --token ${{ secrets.ZITI_CI_GH_TOKEN }}
```

Comment:
Are these the automated dep updates that are attempted?

Seems a bit sad we have to manually specify these when we add a new repo to the project. Adding a new repo seems high friction.

Reply:
yeah. we could theoretically download the ziti go.mod, build a dependency graph and trace it backwards and find the closest roots that didn't recur, but since this doesn't change that often, not sure it's worth it. Would be fun, though.